pygramed/__init__.py

```diff
@@ -53,7 +54,8 @@ def _build_url(cls, url: str, **kwargs: dict):
         header = {"User:Agent": random.choice(USER_AGENTS)}
         async with ClientSession() as session:
             async with session.get(url=url, headers=header) as response:
-                return await response.json(encoding='utf8')
+                res = await response.json(encoding='utf-8')
+                return [Product.create_from_json(x) for x in res]
```

Comment:
Yes :+1: 
You can also call `.__dict__` on each object to deserialize it again. From my point of view it's always better to work with objects, with attributes, with default values rather than with dictionaries full of key errors :laughing: 

The list of attributes is in question. I found a [product](https://www.gramedia.com/api/products/standard-liteme-square/) with images I'll push updated version soon.

What you think about `datetime` on `Format`? I think datetime would be better instead of string.
Cheers :cocktail:

Reply:
Nice, I think it will be good if also handling `datetime` format 👍